src/components/ADempiere/FieldDefinition/FieldNumber.vue

```diff
@@ -197,7 +197,7 @@ export default {
 
       const value = this.$store.getters.getValueOfField({
```

Comment:
Please validate first with `containerUuid` and validate after with `firstTabUuid`.